js/iframe-coordinator/libs/specs/client.spec.ts

```diff
@@ -0,0 +1,245 @@
+import * as ClientInjector from 'inject-loader!../client';
+
+describe("client", () => {
+  let client: any,
```

Comment:
I want to explore mocking in another pull-request since I don't like having these be any-typed.